ophyd/utils/epics_pvs.py

```diff
@@ -20,6 +20,8 @@
 
 from .errors import DisconnectedError, OpException
 
+MIN_POLL_TIME = 0.0001
```

Comment:
This might be too generous: Our default poll_time, 0.01, is probably a more reasonable lower bound. But given that we back off logarithmically, there's probably no harm is putting a lower value as the hard-coded lower bound.